storybook/src/Models/PermissionsModel.qml

```diff
@@ -188,27 +188,15 @@ QtObject {
     function createChannelsModel1() {
         return [
                     {
-                        key: "general",
-                        text: "#general",
-                        color: "lightgreen",
-                        emoji: "👋"
+                        key: "_welcome"
                     },
                     {
-                        key: "faq",
-                        text: "#faq",
-                        color: "lightblue",
-                        emoji: "⚽"
+                        key: "_general"
                     }
                 ]
     }
 
     function createChannelsModel2() {
-        return [
-                    {
-                        key: "socks",
-                        iconSource: ModelsData.icons.socks,
-                        text: "Socks"
-                    }
-                ]
+        return []
```

Comment:
Empty model?

Reply:
yes, in this case community icon/name should be displayed